### PR TITLE
Feature - Export all types & request/response interfaces

### PR DESCRIPTION
**Pull request by e1himself**
It's nice to have access to these interfaces when writing consuming libraries.

Like Alex has it here for Belga syncing code:
https://github.com/prezly/belga-sync/blob/247774b69ba60d7380af62937b3cfbd3a1b6b875/src/util/belga-importer.ts#L5

```ts
import { CoverageCreateRequest } from '@prezly/sdk/dist/Sdk/Coverage/types';
```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1 +1,18 @@
-export { default } from './Sdk';
+export { default } from './Sdk/Sdk';
```
Comment (kudlajz) [inline src/index.ts]: I think this can stay
```suggestion
export { default } from './Sdk';
```

no?

File: src/index.ts [hunk 2]
```diff
@@ -1 +1,18 @@
...
+} from './types'
+export {
+    Options as ClientOptions,
+} from './Sdk/types'
+export {
+    CoverageCreateRequest,
+    CoverageListResponse,
+    CoverageSearchOptions,
+    CoverageUpdateRequest,
+} from './Sdk/Coverage/types'
```
Comment (kamilmielnik) [inline src/index.ts]: I don't think we should go this "deep" - that's not what we do in other repos.

I'd expect `Sdk/Coverage/index.ts` to re-export all these from `./types`, and then `Sdk/index.ts` to re-export all these from `./Coverage`.

I imagine this (`src/index/.ts`) file like this:
```ts
export {
    default,
    CoverageCreateRequest,
    CoverageListResponse,
    CoverageSearchOptions,
    CoverageUpdateRequest,
    Options as ClientOptions,
} from './Sdk';
export {
    Contact,
    Coverage,
    Entity,
    Pagination,
    RoomRef,
    Story,
} from './types';
```

File: src/index.ts [hunk 3]
```diff
@@ -1 +1,18 @@
-export { default } from './Sdk';
+export { default } from './Sdk/Sdk';
+export {
+    Contact,
+    Coverage,
+    Entity,
+    Pagination,
+    RoomRef,
+    Story,
+} from './types'
```
Comment (kamilmielnik) [inline src/index.ts]:
```suggestion
} from './types';
```

The build does not fail - it looks like we're not enforcing `prettier` in this project in any way, even though we have `.prettierrc` and appropriate script in `package.json`.

**Comment by e1himself** [comment]
Thanks for your reviews guys! :+1:

**Pull request merged**